packages/cadl-python/src/emitter.ts

```diff
@@ -406,6 +406,13 @@ function emitParameter(
         implementation: implementation,
         skipUrlEncoding: parameter.type === "endpointPath",
     };
+    if (type.type == "list" && (parameter.type == "query" || parameter.type == "header")) {
```

Comment:
lookds good, can you add a changelog next time? Thanks!

Reply:
Got it, @iscai-msft, sorry for forgetting it again.

Reply:
no worries, thanks @changlong-liu !